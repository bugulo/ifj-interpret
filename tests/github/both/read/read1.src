.IPPcode21

DEFVAR GF@a

READ GF@a int
WRITE GF@a
TYPE GF@a GF@a
WRITE GF@a

CREATEFRAME
DEFVAR TF@a

READ TF@a int
WRITE TF@a
TYPE TF@a TF@a
WRITE TF@a

PUSHFRAME

READ LF@a int
WRITE LF@a
TYPE LF@a LF@a
WRITE LF@a
