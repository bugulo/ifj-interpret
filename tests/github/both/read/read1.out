123int456int789int